pom.xml

```diff
@@ -1341,7 +1327,7 @@
       <dependency>
         <groupId>org.apache.geronimo.mail</groupId>
         <artifactId>geronimo-mail_2.1_provider</artifactId>
-        <version>${geronimo-mail_2.1_provider.version}</version>
+        <version>${version.geronimo-javamail_1.6_mail}</version>
```

Comment:
This version string doesn't look right? Might explain my confusion regarding one of the dropped properties.

Reply:
The name was incorrect, my mistake. now corrected